serde-generate/README.md

```diff
@@ -16,6 +16,7 @@ It can be used as a library or as a command-line tool (see `serdegen` below).
 The following programming languages are fully supported as target languages:
 
 * C++ 17
+* Dart >= 2.14
```

Comment:
It's probably a bit early to claim that Dart is fully supported (see comments on unit tests). Do you mind moving this to the next section with TypeScript 4 ?